README.md

```diff
@@ -1,3 +1,4 @@
+[![Stories in Ready](https://badge.waffle.io/01org/ciao.png?label=ready&title=Ready)](https://waffle.io/01org/ciao)
```

Comment:
@waffle-iron Please group the waffle badges together with the other ciao badges.